float32 actuatorPos # position of the actuator in encoder counts
float32 actuatorVel # velocity of the actuator in counts/s
uint16 actuatorCurrent # current drawn by the actuator
float32 elasticDisplacement # displacement of the SE element in encoder counts
float32 contractileDisplacement # displacement of the CE element
float32 elasticVel # velocity of the elastic element in encoder counts
